Feature/inline external to bootstrap

ashishnagpal2498 opened the pull request:
# Changed Inline and external Css to Bootstrap whichever was possible.

Fixes #122  (issue)

## UI Enhancement

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

## Reviewer: [Vinit Shahdeo](https://github.com/vinitshahdeo)

ashishnagpal2498 commented:
@vinitshahdeo  , Kindly review this PR. Thank you

jainpawan21 commented:
@ashishnagpal2498 please squash your commits. There are so many redundant commits.

ashishnagpal2498 commented:
Done @jainpawan21 .

jainpawan21 commented:
@ashishnagpal2498 Please make this PR to `frontend` branch

ashishnagpal2498 commented:
Hey @jainpawan21 , I have changed the base of PR as well as resolved the merge conflicts. Kindly review it.

vinitshahdeo commented on react-frontend/src/pages/Home.js:
```diff
@@ -46,30 +46,22 @@ class Home extends Component {
-          </center>
+          <hr/>
+          <p>
+            <center><strong>Made by Vinit Shahdeo</strong></center>
```
@ashishnagpal2498 I dont encourage you to use `<center>` tag.

`<center>` is deprecated now. Please play around css or use [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).

ashishnagpal2498 commented:
Hi @vinitshahdeo , removed the deprecated and some unwanted tags. Also I have corrected the flexible movement of tanks, I would appreciate if you could upgrade the PR difficulty. Thanks

ashishnagpal2498 commented:
@vinitshahdeo , this one.
